README.md

```diff
@@ -55,6 +55,18 @@ tests/              - test cases package
 └── test_routes.py  - test suite for service routes
 ```
 
+## RESTful routes for orders and items
```

Comment:
Given PR #36, the one with API doc template has been merged to master, would you like to merge master to resolve and add corresponding changes?

Reply:
Also, there's conflicts with the master branch.

Reply:
Yes, sure! Let me fix the conflicts and commit again.